@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -127,13 +134,41 @@ void main() {
     float segmentDist = mod(vExtrusionCoord.x, dashSize + gapSize) / dashSize;
     float dashDist = 0.5 - distance(segmentDist, halfSegment);
     float dashWidth = fwidth(dashDist);
-    alpha *= smoothstep(-dashWidth, dashWidth, dashDist);
+    float dashedBlendFactor = smoothstep(-dashWidth, dashWidth, dashDist);
+
+    #ifndef USE_DASH_ALPHA
```

Comment:
The name `USE_DASH_ALPHA` is poorly related to what it actually does. Maybe `USE_DASH_COLOR`?